src/robot_dart/robot_dart_simu.cpp

```diff
@@ -407,18 +414,48 @@ namespace robot_dart {
 
     simu::GUIData* RobotDARTSimu::gui_data() { return &(*_gui_data); }
 
-    void RobotDARTSimu::enable_summary_text(bool enable)
+
+    void RobotDARTSimu::enable_text_panel(bool enable) { _enable(_text_panel, enable); }
+    void RobotDARTSimu::enable_status_bar(bool enable) { _enable(_status_bar, enable); }
+    void RobotDARTSimu::_enable(std::shared_ptr<simu::TextData>& text, bool enable)
     {
-        if (!_summary_text && enable) {
-            _summary_text = _gui_data->add_text("Sim. Time: 0s");
+        if (!text && enable) {
+            text = _gui_data->add_text("");
         }
         else if (!enable) {
-            if (_summary_text)
-                _gui_data->remove_text(_summary_text);
-            _summary_text = nullptr;
+            if (text)
+                _gui_data->remove_text(text);
+            text = nullptr;
         }
     }
+    
+    void RobotDARTSimu::set_text_panel(const std::string& str) {
+        if (_text_panel)
+            _text_panel->text = str;
+    }
+
+    void RobotDARTSimu::set_status_bar(const std::string& str) {
+        if (_status_bar)
+            _status_bar->text = str;
+    }
+    
+    std::string RobotDARTSimu::default_text_panel() const {
+        std::ostringstream out;
+        out.precision(3);
+        double rt = _scheduler.current_time();
```

Comment:
`_scheduler.current_time()` does not give the real time passed. At the moment, only `_scheduler.step()` returns this.

Reply:
Should we update the scheduler, so?

Reply:
There's no other way to know the real-time factor. We need to call `scheduler.step()`; we could save this value though to be able to access it..

Reply:
Yes, this is what I had in mind. I will do it.